docs/get-started.md

```diff
@@ -147,6 +147,28 @@ gittuf allows for verifying rules for Git references and files.
 $ gittuf verify-ref --verbose main
 ```
 
+## Verify gittuf itself
```

Comment:
@patzielinski @neilnaveen could you sync so we get effectively this text all in one place, i.e., in dogfood.md? I left a comment talking about essentially adding something like this in #393. Here, we can just point to that one at the end.